packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.0.1]
+
+### Fixed
+
+- No longer add domains that have been granted permissions to `domains` state (nor create for it a selected network proxy) unless the `useRequestQueuePreference` flag is true ([#4368](https://github.com/MetaMask/core/pull/4368))
```

Comment:
```suggestion
- No longer add domains that have been granted permissions to `domains` state (nor create a selected network proxy for it) unless the `useRequestQueuePreference` flag is true ([#4368](https://github.com/MetaMask/core/pull/4368))
```

Reply:
not important. feel free to ignore

Reply:
I think both ways work haha but we can go with yours